litestar/exceptions/http_exceptions.py

```diff
@@ -97,6 +99,12 @@ class ClientException(HTTPException):
     status_code: int = HTTP_400_BAD_REQUEST
 
 
+class ClientDisconnectException(HTTPException):
```

Comment:
This is not a client exception. These exceptions are meant to be raised by library/user code and turned into HTTP error responses. Since this exception is raised after the connection is closed, it cannot be turned into an error response anymore